.ci/eslint-plugin-zotero-translator/lib/rules/translator-id.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const uuid = require('uuid/v4');
+
+const translators = require('../translators').cache;
+
+const deleted = new Set(
+	fs.readFileSync(path.join(translators.repo, 'deleted.txt'), 'utf-8')
+		.split('\n')
+		.map(line => line.split(' ')[0])
+		.filter(id => id && id.indexOf('-') > 0)
+);
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'disallows translatorID re-use',
+			category: 'Potential Problems',
+		},
+		fixable: 'code',
+	},
+
+	create: function (context) {
+		return {
+			Program: function (node) {
+				const filename = context.getFilename();
+				const header = translators.getHeaderFromAST(node);
+				if (!header.declaration) return;
+
+				if (!header.properties.translatorID) {
+					context.report({
+						node: header.declaration,
+						message: 'Header has no translator ID',
+						fix: function (fixer) {
+							const comma = (Object.keys(header.properties).length) ? ',' : '';
+							const sourceCode = context.getSourceCode();
+							return fixer.insertTextAfter(sourceCode.getFirstToken(header.body, t => t.value === '{'), `\n\t"translatorID": "${uuid()}"${comma}`);
+						}
```

Comment:
Same as `lastUpdated` — we shouldn't auto-fix this.

Reply:
Done.